truss/tests/test_control_truss_patching.py

```diff
@@ -0,0 +1,310 @@
+import subprocess
+
+import pytest
+from truss.tests.conftest import CUSTOM_MODEL_USING_EXTERNAL_PACKAGE_CODE
+from truss.tests.test_testing_utilities_for_other_tests import ensure_kill_all
+from truss.tests.test_truss_handle import (
+    verify_environment_variable_on_container,
+    verify_python_requirement_installed_on_container,
+    verify_python_requirement_not_installed_on_container,
+    verify_system_package_installed_on_container,
+    verify_system_requirement_not_installed_on_container,
+)
+from truss.truss_handle import TrussHandle
+
+
+def current_num_docker_images(th: TrussHandle) -> int:
+    return len(th.get_all_docker_images())
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "binary, python_version",
+    [
+        (binary, python_version)
+        for binary in [True, False]
+        for python_version in ["3.8", "3.9"]
```

Comment:
should we increase to match all the supported versions now? I wonder if SUPPORTED_PYTHON_VERSIONS deserves to be its own constant somewhere

Reply:
sure I can make this a constant